### PR TITLE
Some scenarios

### PR DESCRIPTION
**Pull request by maurovallati**
Added a few scenarios used for teaching and / or ICKEPS

### CODE REVIEW

**Review by haz — APPROVED** [review]
These look perfect, thanks!

**Comment by haz** [comment]
@jan-dolejsi : You alright with a merge?

File: assignments/README.md [hunk 1]
```diff
@@ -5,3 +5,12 @@ This space is for student assignments. It should not contain the solutions for o
 ## Propositional
 
 - [Cup of tea for grandpa](cup_of_tea/exercise.md)
```
Comment (jan-dolejsi) [inline assignments/README.md:7]: Was this line removed intentionally?

**Review by jan-dolejsi — COMMENTED** [review]
Great addition.

Just one merge issue in the assignments readme file.

**Comment by maurovallati** [comment]
Oh, my mistake! 

Sent from Yahoo Mail on Android 
 
  On Mon, 25 Jul 2022 at 22:51, Jan ***@***.***> wrote:   


@jan-dolejsi commented on this pull request.

Great addition.

Just one merge issue in the assignments readme file.

In assignments/README.md:
> @@ -5,3 +5,12 @@ This space is for student assignments. It should not contain the solutions for o
 ## Propositional
 
 - [Cup of tea for grandpa](cup_of_tea/exercise.md)

Was this line removed intentionally?

File: assignments/README.md (hunk 1 above)
Comment (haz) [reply assignments/README.md:7]: I think it's actually fine -- the `-` symbol is just part of the markdown, and not an actual line removal.

Comment (jan-dolejsi) [reply assignments/README.md:7]: ha! I was browsing it on my phone and missed that detail.

**Review by jan-dolejsi — APPROVED** [review]
Great contribution.

**Comment by haz** [comment]
It's really not clear! I kept going through the commits to try and find the offending one :P

**Pull request merged**